src/PointwiseFunctions/GeneralRelativity/Psi4.cpp

```diff
@@ -69,27 +71,62 @@ void psi_4(const gsl::not_null<Scalar<ComplexDataVector>*> psi_4_result,
       inverse_spatial_metric, cov_deriv_extrinsic_curvature, r_hat,
       inverse_projection_tensor, projection_tensor, projection_up_lo, 1.0);
 
-  auto& x_coord = get<::Tags::TempI<0, 3, Frame>>(temp_buffer);
+  // Gram-Schmidt x_hat
```

Comment:
Clarify comment: state something like `Gram-Schmidt x_hat, i.e. make x_hat a unit vector orthogonal to r_hat`